<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DELL" name="RetryMulitpleFailedTestCases" tests="5" failures="1" timestamp="29 Jan 2018 14:25:08 GMT" time="218.304" errors="0">
  <testcase name="Logintest" time="28.805" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="ValidateCRMLogo" time="0.132" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="Logintitle" time="0.037" classname="com.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- Logintitle -->
  <testcase name="Logintitle" time="0.029" classname="com.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- Logintitle -->
  <testcase name="Logintitle" time="0.022" classname="com.qa.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [#1 Free CRM for Any Business: Online Customer Relationship Software] but found [#1 Free CRM for Any Business: Online Customer Relationship]">
      <![CDATA[java.lang.AssertionError: expected [#1 Free CRM for Any Business: Online Customer Relationship Software] but found [#1 Free CRM for Any Business: Online Customer Relationship]
	at com.qa.testcases.LoginPageTest.Logintitle(LoginPageTest.java:37)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Logintitle -->
</testsuite> <!-- RetryMulitpleFailedTestCases -->
